### PR TITLE
Antiatomic Fixes

### PR DESCRIPTION
**Pull request by lidevelopers**
I have tested this, and it works fine!

### CODE REVIEW

File: variants.ini [hunk 1]
```diff
@@ -8,7 +8,7 @@ capturesToHand = true
 startFen = rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1
 mustCapture = true
 
-[antiatomic]
+[anti_atomic:giveaway]
```
Comment (TheYoBots) [inline variants.ini:11]: This neither follows giveaway nor atomic.

Comment (TheYoBots) [reply variants.ini:11]: https://github.com/ianfab/Fairy-Stockfish/blob/c96acb864bd52fb440a0ae280010d1259e40dfb2/src/variants.ini#L268-L270

Comment (TheYoBots) [reply variants.ini:11]: When there is a variant existing in fairy sf why take the effort to do extra unneeded stuff

Comment (SriMethan) [reply variants.ini:11]: if possible send a pr please as said in https://github.com/RaviharaV-bot/Liantichess/pull/9#issuecomment-1019037382

**Comment by TheYoBots** [comment]
Why merged?

**Comment by SriMethan** [comment]
> Why merged?

I've tested this and it works so till now we'll keep it, and will fix the unwanted stuffs

**Pull request merged**